eng/Version.Details.xml

```diff
@@ -123,9 +123,9 @@
       <SourceBuild RepoName="linker" ManagedOnly="true" />
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.0.0-4.21404.8" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.0.0-4.21404.3" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@sfoslund Do we need this downgrade?

Reply:
Do we know where it came from?

Reply:
Looks like roslyn

Reply:
cc @mmitche

Reply:
Checking. It looks like maybe it fell back to old style coherency here

Reply:
Pushed a coherency update. It did do something odd here. My hunch is that the strict coherency failed (though no idea why, because it succeeded locally) and a fallback to legacy coherency happened.

Oddly enough though, I would have still expected that strict coherency would have succeeded at one of these commits and pushed the version back to .8.

https://github.com/dotnet/arcade/issues/7720